### PR TITLE
CKS-303 Add GTM container

### PR DESCRIPTION
**Pull request by ediblecode**
https://nicedigital.atlassian.net/browse/CKS-303

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -43,6 +43,34 @@
...
+      "presentation": {
+        "reveal": "always",
+        "panel": "dedicated"
+      }
+    },
+    {
+      "label": "Functional tests - current feature",
+      "detail": "Runs the functional tests for the currently open feature file",
+      "type": "shell",
+      "command": "npm test -- --spec ${fileBasenameNoExtension}",
```
Comment (chunter) [inline .vscode/tasks.json:65]: I dont know npm very well, wondered why there are two sets of --

File: gatsby/gatsby-browser.js [hunk 2]
```diff
@@ -1 +1,15 @@
...
+
+// Gatsby hook for when the route has changed on the client side
+// See https://www.gatsbyjs.org/docs/browser-apis/#onRouteUpdate
+export const onRouteUpdate = ({ prevLocation }) => {
+	if (prevLocation) {
+		// Push our own event to the dataLayer on page change rather than using the
+		// 'gatsby-route-change' event built into gatsby-plugin-google-tagmanager.
+		// Because gatsby-route-change is pushed on initial page load as well as route change.
+		setTimeout(() => {
+			// Slight dealy ensures the title has properly been changed (via react-helmet)
```
Comment (chunter) [inline gatsby/gatsby-browser.js:11]: typo

File: web-app/CKS.Web/Startup.cs [hunk 3]
```diff
@@ -51,5 +53,20 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 endpoints.MapControllers();
             });
         }
+
```
Comment (chunter) [inline web-app/CKS.Web/Startup.cs:56]: Whitespace problem: fucntion seems to be out of line from the one above

File: .vscode/tasks.json (hunk 1 above)
Comment (ediblecode) [reply .vscode/tasks.json:65]: See https://docs.npmjs.com/cli/run-script#description - it's for passing in parameters to child scripts

Comment (ediblecode) [reply .vscode/tasks.json:65]: On a vagutely related note f you get the new cross platform powershell (7) it colour codes tasks nicely so you can see args etc which is kinda nice

File: web-app/CKS.Web/Startup.cs (hunk 3 above)
Comment (ediblecode) [reply web-app/CKS.Web/Startup.cs:56]: I've fixed it in this branch, but there's some changes to C# formatting in https://github.com/nice-digital/cks-gatsby/pull/24/files so I wouldn't worry too much here

**Pull request merged**